BUILD

```diff
@@ -1159,16 +1174,22 @@ tfrt_cc_library(
         "lib/io/buffered_input_stream.cc",
         "lib/io/file_input_stream.cc",
         "lib/io/file_system.cc",
-        "lib/io/posix_file_system.cc",
-        "lib/io/posix_file_system.h",
-    ],
+    ] + select({
+        "@tf_runtime//:windows": [
```

Comment:
would that work ok when using TFRT in a "parent" workspace? (ex. TF) 

I see most place use the fully qualified `@tf_runtime` and I tried to do the same. If I aimed for the wrong consistency please let me know and I'll make the changes.

Reply:
It's complicated.

`@tf_runtime//...` should be used for absolute paths in .bzl macros/rules that are called from dependent repositories. That breaks though if TFRT is not imported as `tf_runtime`. We don't support that for TFRT (but we do for TF), so it's fine.

The slightly clean way is to use `Label()`. This implicitly refers to the repository it was called from.

For BUILD files or macros that are not called from dependent repositories, `//...` is fine for absolute paths. There are many unnecessary `@tf_runtime` prefixes because copybara does those rewrites on export and doesn't know when the explicit repo is necessary.

References to same-package targets really should just use `:foo`.

Reply:
Thanks for the explanation, fixed.